pkg/webhooks/machine_webhook.go

```diff
@@ -859,6 +865,21 @@ func validateAzure(m *machinev1.Machine, config *admissionConfig) (bool, []strin
 		errs = append(errs, field.Invalid(field.NewPath("providerSpec", "osDisk", "diskSizeGB"), providerSpec.OSDisk.DiskSizeGB, "diskSizeGB must be greater than zero and less than 32768"))
 	}
 
+	if providerSpec.OSDisk.DiskSettings.EphemeralStorageLocation != azureEphemeralStorageLocationLocal && providerSpec.OSDisk.DiskSettings.EphemeralStorageLocation != "" {
+		errs = append(errs, field.Invalid(field.NewPath("providerSpec", "osDisk", "diskSettings", "ephemeralStorageLocation"), providerSpec.OSDisk.DiskSettings.EphemeralStorageLocation,
+			fmt.Sprintf("osDisk.diskSettings.ephemeralStorageLocation can be either empty or set to %s", azureEphemeralStorageLocationLocal)))
```

Comment:
This wording update makes it fit the same style as the comments in the openshift API repo
```suggestion
			fmt.Sprintf("osDisk.diskSettings.ephemeralStorageLocation can either be omitted or set to %s", azureEphemeralStorageLocationLocal)))
```